ee/clickhouse/client.py

```diff
@@ -88,7 +88,7 @@ def sync_execute(query, args=None):
 
 
 def format_sql(sql, params):
-    sql = ch_client.substitute_params(sql, params)
+    sql = ch_client.substitute_params(sql, params or {})
```

Comment:
NIT: could add the default to the parameter to avoid more inline logic

Reply:
the default would not override the None passed from sync_execute